shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/api/impl/RuleAlteredJobAPIImpl.java

```diff
@@ -168,7 +168,7 @@ public void stopClusterWriteDB(final String jobId) {
     @Override
     public void stopClusterWriteDB(final String schemaName, final String jobId) {
         LockContext lockContext = PipelineContext.getContextManager().getLockContext();
-        ShardingSphereLock lock = lockContext.getSchemaLock(schemaName).orElse(lockContext.createSchemaLock(schemaName).orElse(null));
+        ShardingSphereLock lock = lockContext.getSchemaLock(schemaName).orElse(lockContext.getSchemaLock(schemaName).orElse(null));
```

Comment:
Seems one `getSchemaLock` call is enough.

Reply:
I will fix

Reply:
I will fix it.